programs/autocrat/src/lib.rs

```diff
@@ -41,22 +41,27 @@ pub mod autocrat {
     }
 
     // ==== autocrat
+    // can anyone add instructions to anyone else's `ProposalInstructions` account?
+    // that could be a security problem
     pub fn create_proposal_instructions(ctx: Context<CreateProposalInstructions>, instructions: Vec<ProposalInstruction>) -> Result<()> {
         instructions::autocrat::create_proposal_instructions::handler(ctx, instructions)
     }
-
+    
     pub fn add_proposal_instructions(ctx: Context<AddProposalInstructions>, instructions: Vec<ProposalInstruction>) -> Result<()> {
         instructions::autocrat::add_proposal_instructions::handler(ctx, instructions)
     }
 
+    // why two parts?
     pub fn create_proposal_part_one(ctx: Context<CreateProposalPartOne>, description_url: String) -> Result<()> {
         instructions::autocrat::create_proposal_part_one::handler(ctx, description_url)
     }
 
+    // floats? floats are pretty scary
```

Comment:
I would like to understand why people feel that way in all cases. This is initializing the price of a pool. So $5.0 seems the same as $5.000001

Reply:
I'll just use integers everywhere, nbd